src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -78,6 +78,21 @@ function IOURequestStepConfirmation({
     const isSharingTrackExpense = action === CONST.IOU.ACTION.SHARE;
     const isCategorizingTrackExpense = action === CONST.IOU.ACTION.CATEGORIZE;
     const isSubmittingFromTrackExpense = action === CONST.IOU.ACTION.SUBMIT;
+    const payeePersonalDetails = useMemo(() => {
+        if (personalDetails?.[transaction?.splitPayerAccountIDs?.[0] ?? -1]) {
+            return personalDetails?.[transaction?.splitPayerAccountIDs?.[0] ?? -1];
+        }
+
+        const participant = transaction?.participants?.find((val) => val.accountID === (transaction?.splitPayerAccountIDs?.[0] ?? -1));
+
+        return {
+            login: participant?.login ?? '',
+            accountID: participant?.accountID ?? -1,
+            avatar: Expensicons.FallbackAvatar,
```

Comment:
shouldn't they have a default avatar created for them from [here](https://github.com/Expensify/App/blob/058044b0baa307afb5325c8ecdcd7cae9357d03e/src/libs/OptionsListUtils.ts#L1867)? if not, let's do `UserUtils.getDefaultAvatar(accountID)` instead

Reply:
ah okay